CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## Improved
+
+- Urls like `/project/node_modules` and `/project/<package name>/node_modules` are now using node package resolve to serve files. #TINY-10281
```

Comment:
I was thinking a minor since existing tests should be unaffected we are not breaking anything since the only thing new is that it would not 404 if the node_modules files are not in the project directory but a parent dir. So it would only be breaking if we have a test that specifically needs to produce a 404 for a node_modules file. Seems like an odd case. 😊

Reply:
Ah I thought this broke compatibility. Carry on 😂